test/org/sagebionetworks/bridge/validators/SurveySaveValidatorTest.java

```diff
@@ -881,11 +881,11 @@ public void operatorsInInfoScreenRulesAreValidated() {
         info.setPromptDetail("prompt detail");
         info.setIdentifier("identifier");
         info.setGuid("guid");
-        info.setRules(Lists.newArrayList(
+        info.setAfterRules(Lists.newArrayList(
                 new SurveyRule.Builder().withValue("foo").withOperator(Operator.EQ).withEndSurvey(true).build()));
```

Comment:
I've verified that in the three ways we validate afterRules, we validate beforeRules, I have not validated the every single validation works through both code paths because I have verified that the same code is executed on both (as both are a list of rules).